src/Servers/Kestrel/Core/src/Internal/KestrelServerImpl.cs

```diff
@@ -160,24 +160,6 @@ private static ServiceContext CreateServiceContext(IOptions<KestrelServerOptions
 
                 async Task OnBind(ListenOptions options, CancellationToken onBindCancellationToken)
                 {
-                    // INVESTIGATE: For some reason, MsQuic needs to bind before
-                    // sockets for it to successfully listen. It also seems racy.
```

Comment:
I haven't noticed any probelsm, and MsQuic no longer needs to bind first. There are now tests that bind both so we'll see if they are flaky or not.